### PR TITLE
layer active when only one selected

### PR DESCRIPTION
**Pull request by sofroniewn**
# Description
This PR forces the layer controls on the left hand side to only directly modify the layer to which they belong. This will make removing the `viewer` easier from an individual `layer` as now there is no way for a layer to go in and modify many layers when those elements are changed. If multiple layers are now selected the control panel on the left disappears, even if those layers are all of the same type. This PR also removes the functionality that allowed multiple clims to be simultaneously adjusted when multiple layers were selected. In practice I have found this functionality confusing and not very helpful. If others (@royerloic) feel strongly that we should support this in some way we can discuss how best to, but the current solution of allowing one layer direct access to all the other layers runs very much against the refactoring that we were trying to achieve.

I may still need to edit / improve some doc strings, but otherwise this is good to go. 

## Type of change

- [x] Bug-fix (non-breaking change which fixes an issue)

# How has this been tested?

- [x] run `example/layers.py` now when multiple layers selected the control panel goes away and it is no longer possible to simultaneously adjust the clims of multiple layers

## Final checklist:
- [x] My PR is the minimum possible work for the desired functionality
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: napari/layers/_image_layer/view/controls.py
Comment (kne42) [inline napari/layers/_image_layer/view/controls.py]: PEP8 here and below

File: napari/layers/_image_layer/view/controls.py
Comment (kne42) [inline napari/layers/_image_layer/view/controls.py]: the list brackets are probably unnecessary here

```suggestion
        self.layer.clim = cmin, cmax
```

**Review by kne42 — APPROVED** [review]
LGTM! Approved pending PEP8 fix.

File: napari/layers/_image_layer/view/controls.py
Comment (sofroniewn) [reply napari/layers/_image_layer/view/controls.py]: fixed

File: napari/layers/_image_layer/view/controls.py
Comment (sofroniewn) [reply napari/layers/_image_layer/view/controls.py]: fixed

**Comment by sofroniewn** [comment]
fixed pep issues, i will wait for @royerloic or @jni to review before merging

**Review by jni — APPROVED** [review]
I like it!

#juansshortesteverreview

**Pull request merged**
